.circleci/scripts/config-template.yml

```diff
@@ -143,6 +143,51 @@ jobs: # Each project will have individual jobs for each specific task it has to
                 paths:
                   - speckle-sharp-ci-tools/Installers
 
+  build-connector-new: # Reusable job for basic connectors
+    executor:
+      name: win/default
+      shell: powershell.exe
+    parameters:
+      slnname:
+        type: string
+      dllname:
+        type: string
+      slug:
+        type: string
+        default: ""
+      installer:
+        type: boolean
+        default: false
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Restore << parameters.slnname >>
+          command: nuget restore << parameters.slnname >>/<< parameters.slnname >>.sln
+      - run:
+          name: Build << parameters.slnname >>
+          command: |
+            $tag = if([string]::IsNullOrEmpty($env:CIRCLE_TAG)) { "0.0.0" } else { $env:CIRCLE_TAG }
+            $semver = if($tag.Contains('/')) {$tag.Split("/")[0] } else { $tag }
+            $ver = if($semver.Contains('-')) {$semver.Split("-")[0] } else { $semver }
+            $channel = if($semver.Contains('-')) {$semver.Split("-")[1] } else { "latest" }
+            $version = "$($ver).$($env:WORKFLOW_NUM)"
+            msbuild << parameters.slnname >>/<< parameters.slnname >>.sln /p:Configuration=Release /p:WarningLevel=0 /p:IsDesktopBuild=false /p:AssemblyVersionNumber=$version /p:AssemblyInformationalVersion=$semver /p:Version=$semver  
+            if(-Not [string]::IsNullOrEmpty($env:CIRCLE_TAG)) { New-Item -Force "speckle-sharp-ci-tools/Installers/<< parameters.slug >>/$channel.yml" -ItemType File -Value "version: $semver" }
+          environment:
+            WORKFLOW_NUM: << pipeline.number >>
+
+      - run: # Creates the inno installer
```

Comment:
This is basically the step we need to deal with, as everything else is unique :)

Maybe we just copy it over to the build-connector job, and just do some conditional steps to just run it on it's slug.